fv3gfs-physics/fv3gfs/physics/stencils/update_dwind_phys.py

```diff
@@ -164,13 +164,26 @@ class AGrid2DGridPhysics:
     Fortran name is update_dwinds_phys
     """
 
-    def __init__(self, stencil_factory: StencilFactory, grid, namelist, grid_info):
-        self.grid = grid
+    def __init__(
+        self,
+        stencil_factory: StencilFactory,
+        partitioner: TilePartitioner,
+        rank,
```

Comment:
```suggestion
        rank: int,
```
What's the type hint on `namelist`? It can't be `Namelist` since that will be owned by `pace.driver`.

Reply:
We need to make config classes for Physics, so this is the old global `spec.namelist`. Need another PR for this.